pages/documentation/grid/model/index.md

```diff
@@ -10,150 +10,122 @@ latex: true
 
-**Internal connection**  
-An internal connection is a non-impedant connection between two components in a voltage level.
+Using **bus/breaker** the voltage level connectivity is described with a coarser level of detail. In this case the vertices of the graph are `Buses`, defined explicitly by the user. A `Bus` has an `Id`, and may have a `name`. Each equipment defines the bus or buses to which it is connected. `Switches` can be defined between buses.
 
-#### Bus/breaker topology
-In bus/breaker topology, the voltage level is described with a coarser level of detail. See the sketch below for an example.
-<span style="color:red"> TODO: add sketch of voltage level in bus/breaker topology.</span>
-The topology is then described as a graph structure, where the vertices are buses and the edges are switches.
+Powsybl provides an integrated Topology Processor that allows to automatically obtain a bus/breaker view from a node/breaker definition, and a bus/branch view from a bus/breaker view or definition. It builds the topology views from the open/close status of `Switches`. `Switches` marked as `retained` in the node/breaker level are preserved in the bus/breaker view.
 
-**Bus**
-[![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/Bus.html)<br>
-A bus is a set of equipments connected at the same voltage.
-When an equipment is connected to a bus, in the IIDM descrition it corresponds to a `Terminal` object.
-In IIDM there is thus one `Terminal` per connected equipment.
+The following diagram represents an example voltage level with two busbars separated by a circuit breaker, a transformer connected to one of them and three generators that can connect to any of the two busbars. The three topology levels are shown.
 
-<span class="heading" id="bb-switch"></span>**Switch**
-[![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/Switch.html)<br>
-<span style="color:red"> TODO: explain the difference with node/breaker switches</span>
+![VoltageLevel](img/index/voltage-level.svg){: width="100%" .center-image}
 
-### Injections
+When defining the model, the user has to specify how the different equipment connect to the network. The connectivity level can be different in each voltage level of the model. If the voltage level is built at node/breaker level, the user has to specify a `Node` when adding an equipment to the model. If the user is building using bus/breaker level, the `ConnectableBus` of the equipment must be specified. If the equipment connection to the network is closed, then the attribute `Bus` must be set and must be equal to `ConnectableBus`. Using this information, the model creates a `Terminal` that will be used to manage the point of connection of the equipment to the network.
```

Comment:
I think this is too technical for this section. It could be used in a tutorial that explain how to create a network from scratch?

Reply:
Indeed we already have a topology tutorial, but it has not be moved in the new web site. I think that this section is also useful, but maybe adding some functional description.